wfd/parser/videoformats.cpp

```diff
@@ -127,10 +127,9 @@ std::vector<EnumType> MaskToEnumList(ArgType from, EnumType biggest_value) {
 
   while (copy != 0) {
     if ((copy & 1) != 0) {
-      if (enum_value > static_cast<unsigned>(biggest_value)) {
-        assert(false);
+      if (enum_value > static_cast<unsigned>(biggest_value))
         break;
-      }
```

Comment:
there should be error logging, or at least comment that this is something unexpectable